run_stack.sh

```diff
@@ -184,12 +189,6 @@ fi
 # each built image is tagged with its target env, so they don't collide with
 # each other; in the case of prod, the tag is "latest".
 if [ "${SKIP_BUILD}" -eq 0 ]; then
```

Comment:
Good catch with the comment; I'll update that.

Also, yes, the more I think on it, the more I agree that having it do a `pull` before the build makes sense.

FYI, I'm currently working on getting the pulled images to populate the layer cache, which I'll add to this PR. I want the build process to occur every time so that it can capture local changes that affect the images and rebuild them if so. Most of the time these builds should be very quick, since all the expensive stuff rarely changes and gets done and cached early in the image building process.